src/main/data/vcf.js

```diff
@@ -41,12 +43,28 @@ function extractLocusLine(vcfLine: string): LocusLine {
 
 function extractVariant(vcfLine: string): Variant {
   var parts = vcfLine.split('\t');
+  var frequency = null;
+  if (parts.length>=7){
```

Comment:
As far as I know vcf can contain multiple variants regarding single nucleotide. It can be achieved in two ways:
1) one entry in vcf can correspond to more than one variant, take a look at this entry:
```20 1234567 microsat1 GTC G,GTCT 50 PASS NS=3;DP=9;AA=G GT:GQ:DP 0/1:35:4 0/2:17:2 1/1:40:3```
2) two entries can refer to the same region in the genome (I think this is what you mentioned in the comment)

Regarding the first issue - there is nothing you can do in term of visualization apart from providing information in the popup. We could think about coloring or some fancy way of highlighting the situation, but in my opinion it's not intuitive.

When you have two (or more) overlaying variants I would also suggest to put it in the popup. I will fix my PR to handle this situation (right now you have only first element that match click in the popup).

If the behaviour is not the one user expected I think the best way to go is suggest user to filter data from vcf file and provide filtered results.

Anyway, this reminds me about one other issue: when you present gene variant it should span through the whole modified region. Right now every variant is visualized by rectangle on the first nucleotide only.